citesphere-messages/src/main/java/edu/asu/diging/citesphere/messages/KafkaTopics.java

```diff
@@ -7,5 +7,7 @@ public interface KafkaTopics {
 
     public final static String REFERENCES_EXPORT_TOPIC = "_citesphere_export_references_topic";
     public final static String REFERENCES_EXPORT_DONE_TOPIC = "_citesphere_export_references_done_topic";
+    
+    public final static String COLLECTION_IMPORT_TOPIC = "_citesphere_import_collections_topic";
```

Comment:
Let's add javadoc here explaining what this topic is used for (it's starting to get less intuitively understandable).